src/redux/chromeReducers.ts

```diff
@@ -155,14 +155,6 @@ export function loadModulesSchemaReducer(
   };
 }
 
-export function changeActiveModuleReducer(state: ChromeState, { payload }: { payload: string }): ChromeState {
```

Comment:
It is pretty crazy how much Redux logic can be removed switching to a bottom up state management solution.

Reply:
Yup. And you can use the atoms outside of the react context out of the box.